stdlib/3/typing.pyi

```diff
@@ -418,3 +417,11 @@ class Pattern(Generic[AnyStr]):
 # Functions
 
 def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+
+
+# Type constructors
+
+# NamedTuple is special-cased in the type checker; the initializer is ignored.
+def NamedTuple(typename: str, fields: Iterable[Tuple[str, Any]], *,
+               verbose: bool = ..., rename: bool = ..., module: str = None) -> Type[tuple]: ...
```

Comment:
`fields` can be more exact as `Iterable[Tuple[str, Union[str, Type]]]`  or something like that.